crates/sui-types/src/quorum_driver_types.rs

```diff
@@ -146,17 +131,6 @@ pub struct ExecuteTransactionRequestV3 {
     pub include_auxiliary_data: bool,
 }
 
-#[derive(Clone, Debug)]
-pub struct VerifiedExecuteTransactionResponseV3 {
-    pub effects: VerifiedCertifiedTransactionEffects,
-    pub events: Option<TransactionEvents>,
-    // Input objects will only be populated in the happy path
-    pub input_objects: Option<Vec<Object>>,
-    // Output objects will only be populated in the happy path
-    pub output_objects: Option<Vec<Object>>,
-    pub auxiliary_data: Option<Vec<u8>>,
-}
```

Comment:
I know this is dead code but is it ok to remove?

Reply:
Since there is no reference to this type else where, it should be safe to remove.